workshop/content/docs/publishing/ogcapi-maps.md

```diff
@@ -13,13 +13,14 @@ any geospatial data as a georeferenced map image.
 
 pygeoapi supports the OGC API - Maps draft specification, using [MapServer MapScript](https://www.mapserver.org/mapscript) and a WMS facade as core backends.
 
-To expose data using `MapScript` following configuration can be added 
+In this section we'll be exposing a shapefile available at `/data/airport/airport.shp` location. This data can be consumed with various clients which can access OGC APIs - Maps. Here we can also pass style in *.sld* format. Which can be generated on [Geoserver](https://docs.geoserver.org/stable/en/user/styling/index.html), [QGIS](https://www.qgistutorials.com/en/docs/3/basic_vector_styling.html), etc.
+ 
```

Comment:
Instead of just pasting the code, put something similar to this:
https://github.com/geopython/diving-into-pygeoapi/blob/main/workshop/content/docs/publishing/ogcapi-features.md?plain=1#L42